### ISSUE TITLE
Text situated in a Sidebar component can have screen reader issues

### ISSUE THREAD

**Issue by elvis-iacobescu-sage** [Sage/carbon#6999]
### Description

I am trying to make <sidebar> content accessible to screen readers by tabbing. the sidebar has no buttons only texts and <TileCompoent>.

### Reproduction

https://stackblitz.com/edit/parsium-carbon-starter-euvanf?file=index.html,src%2FApp.tsx

### Steps to reproduce

- open sidebar 
- Tabb with NVMA started
- NVMA does not read content 

### Carbon version

142.9.0

### Relevant browsers

Firefox, Chrome, Safari, Microsoft Edge, Other

### Relevant OSs

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nineteen88** [comment]
FE-6839

**Comment by tomdavies73** [comment]
The issue is closed due to the text content still being accessible when navigated using VO keys, opposed to tabbing. With the former being the preferred and recommended form of navigation for screen reader users. Verified by the accessibility team.
